tests/trace/util.py

```diff
@@ -67,3 +68,22 @@ def __eq__(self, other):
 
 class DummyTestException(Exception):
     pass
+
+
+def get_clickhouse_loglines(
```

Comment:
what makes this clickhouse specific?

Reply:
hmm nothing, other than clickhouse batched uses logger.info. changing